lib/addons/cloudwatch-adot-addon/collector-config-cloudwatch.ytpl

```diff
@@ -311,7 +344,7 @@ spec:
             operations:
               - action: add_label
                 new_label: EKS_Cluster
-                new_value: ${CLUSTER_NAME}
```

Comment:
Why did it deploy at all before, lol.

Reply:
This is was never a problem. Prebiously we used env var name and now we are rendering directly because `"{{clusterName}}"` is rendered in the code.